src/template.html

```diff
@@ -9,11 +9,10 @@
   <body>
     <div class="general-wrapper">
       <h2 class="leaderboard-header">Leaderboard</h2>
```

Comment:
- [ ] [OPTIONAL] Adding a long score can break the UI, It's recommended that you add some kind of validation to block this behavior

Reply:
Ok, noted with thanks.